### ISSUE TITLE
Failed to send shutdown command with windows 2012 on vmware-iso

### ISSUE THREAD

**Issue by moodwrench** [mitchellh/packer#4171]
Version: Packer v0.11.0

we've started seeing an issue that only seems to affect windows 2012 hosts using the vmware-iso builder on a remote ESXi machine 

"type": "vmware-iso"
"remote_type": "esx5"
"shutdown_command": "shutdown /s /t 10 /f /d p:4:1 /c \"Packer Shutdown\"",

It seems that the build completes successfully with no communications issues or winrm issues, then times out connecting for the shutdown.

debug logs:
```
2016/11/15 14:19:18 ui: ==> vmware-iso: Gracefully halting virtual machine...
==> vmware-iso: Gracefully halting virtual machine...
2016/11/15 14:19:18 packer: 2016/11/15 14:19:18 Executing shutdown command: shutdown /s /t 10 /f /d p:4:1 /c "Packer Shutdown"
2016/11/15 14:20:21 ui error: ==> vmware-iso: Failed to send shutdown command: unknown error Post http://xxx.xxx.xxx.xxx:5985/wsman: dial tcp xxx.xxx.xxx.xxx:5985: getsockopt: connection timed out
2016/11/15 14:20:21 packer: 2016/11/15 14:20:21 opening new ssh session
==> vmware-iso: Failed to send shutdown command: unknown error Post http://xxx.xxx.xxx.xxx:5985/wsman: dial tcp xxx.xxx.xxx.xxx:5985: getsockopt: connection timed out
2016/11/15 14:20:21 packer: 2016/11/15 14:20:21 starting remote command: vim-cmd vmsvc/power.getstate 36
2016/11/15 14:20:22 ui: ==> vmware-iso: Destroying virtual machine...
==> vmware-iso: Destroying virtual machine...
2016/11/15 14:20:22 packer: 2016/11/15 14:20:22 opening new ssh session
2016/11/15 14:20:22 packer: 2016/11/15 14:20:22 starting remote command: vim-cmd vmsvc/destroy 36
2016/11/15 14:20:22 packer: 2016/11/15 14:20:22 opening new ssh session
2016/11/15 14:20:22 packer: 2016/11/15 14:20:22 starting remote command: test ! -e /vmfs/volumes/iso_datastore/windows_2012_r2-v1479232908
2016/11/15 14:20:22 packer: 2016/11/15 14:20:22 Remote command exited with '1': test ! -e /vmfs/volumes/iso_datastore/windows_2012_r2-v1479232908
2016/11/15 14:20:22 packer: 2016/11/15 14:20:22 opening new ssh session
2016/11/15 14:20:22 packer: 2016/11/15 14:20:22 starting remote command: test ! -e /vmfs/volumes/iso_datastore/windows_2012_r2-v1479232908
2016/11/15 14:20:22 packer: 2016/11/15 14:20:22 Remote command exited with '1': test ! -e /vmfs/volumes/iso_datastore/windows_2012_r2-v1479232908
2016/11/15 14:20:22 packer: 2016/11/15 14:20:22 opening new ssh session
2016/11/15 14:20:22 packer: 2016/11/15 14:20:22 starting remote command: test ! -e /vmfs/volumes/iso_datastore/windows_2012_r2-v1479232908
2016/11/15 14:20:22 packer: 2016/11/15 14:20:22 Deleting floppy disk: /tmp/packer222785091
2016/11/15 14:20:22 packer: 2016/11/15 14:20:22 opening new ssh session
2016/11/15 14:20:22 packer: 2016/11/15 14:20:22 starting remote command: test -e /vmfs/volumes/iso_datastore/windows_2012_r2-v1479232908
2016/11/15 14:20:23 packer: 2016/11/15 14:20:23 Remote command exited with '1': test -e /vmfs/volumes/iso_datastore/windows_2012_r2-v1479232908
Build 'vmware-iso' errored: Failed to send shutdown command: unknown error Post http://xxx.xxx.xxx.xxx:5985/wsman: dial tcp xxx.xxx.xxx.xxx:5985: getsockopt: connection timed out

==> Some builds didn't complete successfully and had errors:
2016/11/15 14:20:23 ui error: Build 'vmware-iso' errored: Failed to send shutdown command: unknown error Post http://xxx.xxx.xxx.xxx:5985/wsman: dial tcp xxx.xxx.xxx.xxx:5985: getsockopt: connection timed out
2016/11/15 14:20:23 Builds completed. Waiting on interrupt barrier...
2016/11/15 14:20:23 machine readable: error-count []string{"1"}
2016/11/15 14:20:23 ui error: 
==> Some builds didn't complete successfully and had errors:
--> vmware-iso: Failed to send shutdown command: unknown error Post http://xxx.xxx.xxx.xxx:5985/wsman: dial tcp xxx.xxx.xxx.xxx:5985: getsockopt: connection timed out

2016/11/15 14:20:23 machine readable: vmware-iso,error []string{"Failed to send shutdown command: unknown error Post http://xxx.xxx.xxx.xxx:5985/wsman: dial tcp xxx.xxx.xxx.xxx:5985: getsockopt: connection timed out"}
2016/11/15 14:20:23 ui error: --> vmware-iso: Failed to send shutdown command: unknown error Post http://xxx.xxx.xxx.xxx:5985/wsman: dial tcp xxx.xxx.xxx.xxx:5985: getsockopt: connection timed out
2016/11/15 14:20:23 ui: 
==> Builds finished but no artifacts were created.
```

**Comment by rickard-von-essen** [comment]
Seems to be a duplicate of #4041, fixed on master. Please reopen if you think otherwise.

**Comment by moodwrench** [comment]
This issue still exists after upgrading to 0.12
